homeassistant/components/zwave_js/strings.json

```
@@ -47,7 +47,7 @@
           "s2_unauthenticated_key": "S2 Unauthenticated Key",
           "usb_path": "[%key:common::config_flow::data::usb_path%]"
         },
-        "description": "The add-on will generate security keys if those fields are left empty.",
+        "description": "Select your Z-Wave antenna",
```

```suggestion
        "description": "Select your Z-Wave adapter",
```
this matches the description on L132. 
Also, the step has a lot of key-related stuff in addition to the usb_path. Is this description correct for this? Seems less comprehensive than the title.